CONTRIBUTING.md

```diff
@@ -146,25 +133,48 @@ Once your PR is approved (hooray!) the reviewer will squash your commits into a
 single commit, and then merge the commit onto the Go Cloud master branch. Thank
 you!
 
-### Squashing
-
-(This section is for project members.)
-
-When you are squashing-and-merging a pull request into master, please edit the
-commit message to fit in with the [project conventions][commit messages]. In
-particular:
-
--   Ensure that the first line summary starts with the primary package affected,
-    followed by a colon, followed by a short summary of the change, like
-    "blob/gcsblob: made more blobby (#123)".
--   If the change is addressing an issue, make the commit message end with a
-    block of "Updates #123" and/or "Fixes #456", one issue number per line.
--   Don't include the automatically added code-review commit lines like "Fixed
-    some stuff" or "Addressed feedback".
-
-[#317][] and [#318][] are tracking improvements to Contribute Bot to
-automatically warn for some of these problems.
-
-[#317]: https://github.com/google/go-cloud/issues/317
-[#318]: https://github.com/google/go-cloud/issues/318
-[commit messages]: https://golang.org/doc/contribute.html#commit_messages
+## Github code review workflow conventions
+
+(For project members and frequent contributors).
+
+As a contributor:
+
+-   Try hard to make each Pull Request as small and focused as possible.
+-   Adding someone as a Reviewer means "please feel free to look and comment";
+    the review is optional. Choose as many Reviewers as you'd like.
+-   Adding someone as an Assignee means that the Pull Request should not be
+    submitted until they approve. If you choose multiple Assignees, wait until
+    all of them approve. It is fine to ask someone if they are OK with being
+    removed as an Assignee.
+    -   Note that if you don't select any assignees, ContributeBot will turn all
+        of your Reviewers into Assignees.
+-   Make as many commits as you want locally, but try not to push them to Github
+    until you've addressed comments; this allows the email notification about
+    the push to be a signal to reviewers that the PR is ready to be looked at
+    again.
+-   When there may be confusion about what should happen next for a PR, be
+    explicit; add a "PTAL" comment if it is ready for review again, or a "Please
+    hold off on reviewing for now" if you are still working on addressing
+    comments.
+-   "Resolve" comments that you are sure you've addressed; let your reviewers
+    resolve ones that you're not sure about.
+-   Do not use `git push --force`; this can cause comments from your reviewers
```

Comment:
I think this is because of rebasing on current master instead of merging.

Reply:
Added a note about this.